library/init/meta/tactic.lean

```diff
@@ -101,6 +101,11 @@ meta def try_lst : list (tactic unit) → tactic unit
     end
   end
 
+meta def continue (t : tactic α) : tactic (option α) :=
+λ s, result.cases_on (t s)
+ (λ a, success (some a))
+ (λ e ref s', success none s')
+
```

Comment:
We have a tactic called `try_core`
https://github.com/leanprover/lean/blob/master/library/init/meta/tactic.lean#L81-L84